### PR TITLE
fix(curriculum) replace single-line blocks with multi-line blocks for Front End Libraries

### PR DESCRIPTION
**Pull request by LaurentLabine**
Implemented changes for the Front End Certification.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Checks Front End Libraries on #41518

### CODE REVIEW

**Comment by LaurentLabine** [comment]
Alright, just finished with this.  Let me know if you have any feedback or questions!  Thanks

File: curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md
Comment (RandellDawson) [inline curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md:27]: You have an extra `html` that should not be here.
````suggestion
```
````

File: curriculum/challenges/english/03-front-end-libraries/react/introducing-inline-styles.md
Comment (RandellDawson) [inline curriculum/challenges/english/03-front-end-libraries/react/introducing-inline-styles.md:17]:
````suggestion
```jsx
````

File: curriculum/challenges/english/03-front-end-libraries/react/introducing-inline-styles.md
Comment (RandellDawson) [inline curriculum/challenges/english/03-front-end-libraries/react/introducing-inline-styles.md:23]:
````suggestion
```jsx
````

File: curriculum/challenges/english/03-front-end-libraries/react/use--for-a-more-concise-conditional.md
Comment (RandellDawson) [inline curriculum/challenges/english/03-front-end-libraries/react/use--for-a-more-concise-conditional.md:13]:
````suggestion
```jsx
````

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@LaurentLabine  Thank you for your PR. In addition to the requested changes listed below, you are missing inline code block for the following challenge:

- `curriculum/challenges/english/03-front-end-libraries/jquery/remove-classes-from-an-element-with-jquery.md`

**Comment by RandellDawson** [comment]
@LaurentLabine On a separate note, you really should take a look at https://contribute.freecodecamp.org and specifically the [Managing changes locally](https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally)https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally) as you have made a PR from change to your `main` branch.  You want to keep your `main` branch clean and keep it in sync with the upstream repository (freeCodeCamp) and create **new** branches off `main` to make commits to and create PRs from.

File: curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md
Comment (Sembauke) [reply curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md:27]: Tip in the file tab on GitHub you can batch all these suggestions together in one single commit.

**Comment by LaurentLabine** [comment]
Hey,

Thank you for your review and your time and sorry for the mistakes I will correct them and re-submit.

I also was under the impression that either ways worked as far as making a branch on the product itself or making my changes in a forked version before pulling back.  Thank you I will go through the article you linked!

**Review by LaurentLabine — COMMENTED** [review]
Fixed mistake as React outputs JSX and not JS

File: curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md
Comment (RandellDawson) [reply curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md:27]: @LaurentLabine You have still not resolved the issue above. Also, you have still not made a change to the file I referenced in my first comment.

Comment (LaurentLabine) [reply curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md]: Hey, Yeah I' still working on it, sorry!  I am not sure how to manage my current pull request and additional changes that must be added... Should I close this one and re-submit a new one with everything sorted out?

Comment (LaurentLabine) [reply curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md]: I just made the changes on my local branch and was actually wondering how to proceed next wether I could add the missing changes to my current PR, make a new one for the missing file and html correction or whether I should re do it all together closing the initial request we've been working on...

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
LGTM

File: curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md
Comment (nhcarrigan) [reply curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md]: Hey there,

If you have changes prepared locally, you should be able to commit them and `git push` up to your remote repository - this PR would then automatically update. 

For the changes Randy has suggested with the GitHub UI here, like Sem mentioned you can switch to the file view and commit them in a batch through the UI - do note that you should do this after you've pushed up your local changes, to avoid conflicts.

Comment (LaurentLabine) [reply curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md]: Hey!  Yeah it got me confused but that is what I understood happened...  My understanding is that it's been integrated and everything about this PR has been dealt with now.. Right?

Comment (nhcarrigan) [reply curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md]: I am still seeing @RandellDawson's suggestions as outstanding.

**Review by LaurentLabine — COMMENTED** [review]
Reviewed and made changes

File: curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md
Comment (LaurentLabine) [reply curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md]: I believe the changes are in and got one approval and the system is waiting for a second one in order to merge the changes?

I find it pretty confusing but I did make the changes and from what I can see the modifications have been integrated in the PR?

Two of the changes I edited and uploaded locally to my branch which I believe as you brought up have been automatically taken into account in the initial PR and the other ones I merged them from the recommendations that were made during the review so that might be why I still have some outstanding comments left hanging?

File: curriculum/challenges/english/03-front-end-libraries/react/introducing-inline-styles.md
Comment (RandellDawson) [reply curriculum/challenges/english/03-front-end-libraries/react/introducing-inline-styles.md:23]: @LaurentLabine You have not implemented this requested change, so I am not sure why you marked it Resolved.

File: curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md
Comment (LaurentLabine) [reply curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md]: Just realised I had the option to set two changes to resolved... Is it better now?

File: curriculum/challenges/english/03-front-end-libraries/react/use--for-a-more-concise-conditional.md
Comment (RandellDawson) [reply curriculum/challenges/english/03-front-end-libraries/react/use--for-a-more-concise-conditional.md:13]: @LaurentLabine You have not implemented this requested change, so I am not sure why you marked it Resolved.

File: curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md
Comment (nhcarrigan) [reply curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md]: I see a commit you made that resolved a couple of changes, yes - but there are some that were not in that commit as well.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
In addition to making two suggested changes, you still also need to fix `curriculum/challenges/english/03-front-end-libraries/jquery/remove-classes-from-an-element-with-jquery.md`.

**Comment by RandellDawson** [comment]
@nhcarrigan Thanks for catching the fact that the suggested changes were not actually commited.

**Comment by LaurentLabine** [comment]
Right...  At this point I have also gone through the documentation you have pointed out and sanitized my main branch by checking it out so I could do things properly so I'm not sure what my best option is to solve this...

**Comment by LaurentLabine** [comment]
The easiest way I can think of is closing this one at this time and re-doing the change and re-doing the whole thing tomorrow..?

**Comment by LaurentLabine** [comment]
Sorry for all the mess.  I just re-did everything on a branch following the guidelines I got from my mistakes here and created a new pull request that I think should be much simpler.  Once again thank you for the feedback and support with this and I hope I am not making a mistake closing this.

Here is a link to the new one: https://github.com/freeCodeCamp/freeCodeCamp/pull/41525

**Pull request closed without merging**
